mmv1/products/datafusion/Instance.yaml

```diff
@@ -35,6 +35,17 @@ timeouts: !ruby/object:Api::Timeouts
   update_minutes: 25
   delete_minutes: 50
 autogen_async: true
+virtual_fields:
+  - !ruby/object:Api::Type::Boolean
+    name: 'deletion_protection'
+    default_value: false
+    description: |
+      Whether Terraform will be prevented from destroying the instance. Defaults to true.
+      When a`terraform destroy` or `terraform apply` would delete the instance,
+      the command will fail if this field is not set to false in Terraform state.
+      When the field is set to true or unset in Terraform state, a `terraform apply`
+      or `terraform destroy` that would delete the instance will fail.
```

Comment:
When the field is set to true in Terraform state, a `terraform apply` or `terraform destroy` that would delete the instance will fail. When the field is set to false or unset, deleting the instance is allowed

Reply:
done